app/views/schools/dashboard/_school_cohort_details.html.erb

```diff
@@ -1,7 +1,7 @@
 <% lead_provider = school_cohort.default_induction_programme&.lead_provider %>
 <% delivery_partner =  school_cohort.default_induction_programme&.delivery_partner %>
 <% ects_count = school_cohort.current_induction_records.count %>
-<% partnership = school_cohort.school.partnerships.active.find_by(cohort: school_cohort.cohort) %>
+<% latest_partnership = school_cohort.school.partnerships.where(cohort: school_cohort.cohort).order(created_at: :desc).limit(1).first %>
```

Comment:
Can you explain this one? I read this and thought about whether ordering by `created_at` enough? Is it possible that you select the wrong partnership here if there are multiple?

Reply:
I don't know how we create partnerships, so I had to assume the latest one will be always the one we need. @tonyheadford may know more?

Reply:
@jamesrose Tony suggested we should be ok if we included `relationship: false` in the `where` statement. I've already pushed it.

Reply:
Thanks! 🚢